examples/deal.II/bp.cc

```diff
@@ -0,0 +1,235 @@
+// ---------------------------------------------------------------------
```

Comment:
minor note - for MFEM, NEK5000, and PETSc we call this file `bps.*` since it can do all 6 BPs